phpBB/includes/compatibility_globals.php

```diff
@@ -44,7 +44,6 @@
 // Grab global variables, re-cache if necessary
 /* @var $config phpbb\config\db */
 $config = $phpbb_container->get('config');
-set_config(null, null, null, $config);
```

Comment:
How about in `set_config()` directly? Adding this after `if ($set_config !== null)` (not sure if it works or not):

```php
if ($config === null)
{
	global $config;
}
```

This question is also valid for `set_config_count()` and `request_var()`.